### ISSUE TITLE
waf breaks with python3.7

### ISSUE THREAD

**Issue by dvzrv** [jackaudio/jack2#366]
On Arch Linux we're now on python3.7.
Waf seems to break with it:

```
Traceback (most recent call last):
  File "/home/dave/Downloads/jack2/waflib/Node.py", line 540, in ant_iter
    raise StopIteration
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dave/Downloads/jack2/waflib/Scripting.py", line 161, in waf_entry_point
    run_commands()
  File "/home/dave/Downloads/jack2/waflib/Scripting.py", line 259, in run_commands
    parse_options()
  File "/home/dave/Downloads/jack2/waflib/Scripting.py", line 211, in parse_options
    Context.create_context('options').execute()
  File "/home/dave/Downloads/jack2/waflib/Options.py", line 271, in execute
    super(OptionsContext, self).execute()
  File "/home/dave/Downloads/jack2/waflib/Context.py", line 226, in execute
    self.recurse([os.path.dirname(g_module.root_path)])
  File "/home/dave/Downloads/jack2/waflib/Context.py", line 304, in recurse
    user_function(self)
  File "/home/dave/Downloads/jack2/wscript", line 387, in options
    opt.load('compiler_cxx')
  File "/home/dave/Downloads/jack2/waflib/Context.py", line 219, in load
    fun(self)
  File "/home/dave/Downloads/jack2/waflib/Tools/compiler_cxx.py", line 97, in options
    opt.load_special_tools('cxx_*.py')
  File "/home/dave/Downloads/jack2/waflib/Context.py", line 602, in load_special_tools
    lst = self.root.find_node(waf_dir).find_node('waflib/extras').ant_glob(var)
  File "/home/dave/Downloads/jack2/waflib/Node.py", line 631, in ant_glob
    ret = [x for x in self.ant_iter(accept=accept, pats=[to_pat(incl), to_pat(excl)], maxdepth=kw.get('maxdepth', 25), dir=dir, src=src, remove=kw.get('remove', True))]
  File "/home/dave/Downloads/jack2/waflib/Node.py", line 631, in <listcomp>
    ret = [x for x in self.ant_iter(accept=accept, pats=[to_pat(incl), to_pat(excl)], maxdepth=kw.get('maxdepth', 25), dir=dir, src=src, remove=kw.get('remove', True))]
RuntimeError: generator raised StopIteration
```

**Comment by falkTX** [comment]
PR welcome :)

**Comment by falkTX** [comment]
Now fixed

**Comment by renich** [comment]
How was this fixed? I still can't build v1.9.12 with python3.7 in flatpak. 

```
Building module jack2 in /home/renich/Projects/flatpak/bitwig-studio/.flatpak-builder/build/jack2-19
========================================================================
Note: checking out 'v1.9.12'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at c164781 Update ChangeLog
Running: python3 waf configure --prefix=/app --htmldir=/app/share/doc/jack/ --freebob=no --autostart=default
Traceback (most recent call last):
  File "/run/build/jack2/waflib/Node.py", line 540, in ant_iter
    raise StopIteration
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/run/build/jack2/waflib/Scripting.py", line 161, in waf_entry_point
    run_commands()
  File "/run/build/jack2/waflib/Scripting.py", line 259, in run_commands
    parse_options()
  File "/run/build/jack2/waflib/Scripting.py", line 211, in parse_options
    Context.create_context('options').execute()
  File "/run/build/jack2/waflib/Options.py", line 271, in execute
    super(OptionsContext, self).execute()
  File "/run/build/jack2/waflib/Context.py", line 226, in execute
    self.recurse([os.path.dirname(g_module.root_path)])
  File "/run/build/jack2/waflib/Context.py", line 304, in recurse
    user_function(self)
  File "/run/build/jack2/wscript", line 387, in options
    opt.load('compiler_cxx')
  File "/run/build/jack2/waflib/Context.py", line 219, in load
    fun(self)
  File "/run/build/jack2/waflib/Tools/compiler_cxx.py", line 97, in options
    opt.load_special_tools('cxx_*.py')
  File "/run/build/jack2/waflib/Context.py", line 602, in load_special_tools
    lst = self.root.find_node(waf_dir).find_node('waflib/extras').ant_glob(var)
  File "/run/build/jack2/waflib/Node.py", line 631, in ant_glob
    ret = [x for x in self.ant_iter(accept=accept, pats=[to_pat(incl), to_pat(excl)], maxdepth=kw.get('maxdepth', 25), dir=dir, src=src, remove=kw.get('remove', True))]
  File "/run/build/jack2/waflib/Node.py", line 631, in <listcomp>
    ret = [x for x in self.ant_iter(accept=accept, pats=[to_pat(incl), to_pat(excl)], maxdepth=kw.get('maxdepth', 25), dir=dir, src=src, remove=kw.get('remove', True))]
RuntimeError: generator raised StopIteration
Error: module jack2: Child process exited with code 2
make: [Makefile:20: install] Error 1 (ignored)

```

**Comment by renich** [comment]
Nevermind... found it on master.
